src/core/Table/Table.test.tsx

```diff
@@ -1486,3 +1486,97 @@ it('should render sub-rows with custom expander', () => {
   rows = container.querySelectorAll('.iui-table-body .iui-row');
   expect(rows.length).toBe(10);
 });
+
+it('should edit cell data', () => {
+  const onCellEdit = jest.fn();
+  const data = mockedData(4);
+  const columns: Column<TestDataType>[] = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+          isCellEditable: (rowData) => rowData.name !== 'Name2',
+        },
+        {
+          id: 'description',
+          Header: 'description',
+          accessor: 'description',
+        },
+        {
+          id: 'view',
+          Header: 'view',
+          Cell: () => {
+            return <span>View</span>;
+          },
```

Comment:
Huh?

Reply:
Oh ok I see you copied from other test and removed `onClick`. Maybe could remove this entire column as it doesn't do anything for this test.

Reply:
`assertRowsData` expects that there will be `View` column, so I need to add it.